File: app/src/main/java/github/umer0586/sensorserver/activities/DeviceAxisActivity.java
package github.umer0586.sensorserver.activities;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.AppCompatImageView;

import android.graphics.drawable.Drawable;
import android.os.Bundle;

import java.io.IOException;
import java.io.InputStream;

import github.umer0586.sensorserver.R;

public class DeviceAxisActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_device_axis);
        AppCompatImageView deviceAxisImage = findViewById(R.id.device_axis);

        try
        {

            InputStream ims = getAssets().open("axis_device.png");
            Drawable d = Drawable.createFromStream(ims, null);
            deviceAxisImage.setImageDrawable(d);
            ims.close();

        } catch (IOException ex)
        {
            return;
        }

    }
}

File: app/src/main/java/github/umer0586/sensorserver/activities/DeviceAxisActivity.kt
package github.umer0586.sensorserver.activities

import android.graphics.drawable.Drawable
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.AppCompatImageView
import github.umer0586.sensorserver.R
import java.io.IOException

class DeviceAxisActivity : AppCompatActivity()
{


    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_device_axis)
        val deviceAxisImage = findViewById<AppCompatImageView>(R.id.device_axis)
        try
        {
            val ims = assets.open("axis_device.png")
            val d = Drawable.createFromStream(ims, null)
            deviceAxisImage.setImageDrawable(d)
            ims.close()
        }
        catch (ex: IOException)
        {
            return
        }
    }
}